### PR TITLE
Gobstones CLI

### PR DESCRIPTION
**Pull request by afska**
@flbulgarelli 
What do you think about renaming & moving this to `gobstones/gs-weblang-cli`?
Does the `--ast` option cover your needs at Mulang?

Fixes https://github.com/gobstones/gs-weblang-core/issues/20

### CODE REVIEW

**Comment by flbulgarelli** [comment]
> What do you think about renaming & moving this to gobstones/gs-weblang-cli?

Its ok for me

File: README.md [hunk 1]
```diff
@@ -1,15 +1,173 @@
...
+# requires node > 4
+sudo npm install -g gs-weblang-cli
+```
 
 ## Building wrapper
 
+The cli can be wrapped as an executable using [nexe](https://github.com/jaredallard/nexe): `npm install nexe -g`.
+
+```
+git clone https://github.com/gobstones/gs-weblang-cli gobs
```
Comment (flbulgarelli) [inline README.md]: @rodri042 I would rename the executable to name gs-weblang-cli.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,173 @@
...
+```bash
+gobs --batch request.json
+# returns an array with the responses
+```
+
+### Generate AST
+
+```bash
+echo "program {\n Poner(Azul)\n }" > /tmp/gobs.gbs
+gobs --ast /tmp/gobs.gbs
```
Comment (flbulgarelli) [inline README.md]: I really need to avoid writing a file. Can this input be read from `stdin`?

Comment (flbulgarelli) [reply README.md]: I mean, I am not against having a file input option, but default should still be reading from `stdin`

**Review by flbulgarelli — CHANGES_REQUESTED** [review]
Could we add some tests?

I mean, I it is not a hard request, but a nice to have

File: test/run-spec.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+    it("should catch unexpected errors", function() {
+      var output = exec("");
+      output.should.eql({
+        "status": "all_is_broken_error",
+        "message": "Something has gone very wrong",
+        "detail": {},
+        "moreDetail": "Cannot read property 'range' of null"
+      });
+    });
```
Comment (flbulgarelli) [inline test/run-spec.js:101]: You rock!

File: README.md (hunk 2 above)
Comment (afska) [reply README.md]: Done. Now it can be executed like this: `echo "program {\n Poner(Azul)\n }" | gs-weblang-cli --ast --from_stdin`

**Comment by afska** [comment]
I'll close the PR, add a CI and migrate this to https://github.com/gobstones/gs-weblang-cli

**Pull request closed without merging**